its/projects/shared/struts-1.3.9-diet/core/pom.xml

```diff
@@ -25,7 +25,7 @@
     <parent>
         <groupId>org.apache.struts</groupId>
         <artifactId>struts-parent</artifactId>
-        <version>1.3.9</version>
+        <version>1,3,9</version>
```

Comment:
It does not look like we are explicitly testing for the version containing commas. Is this change expected here?

Reply:
I chose to have commas in the version in order to be consistent with the jar having commas since putting commas in the directory/package names would affect all files below.
Anyway, it's probably better to go for the "invoker" test